### PR TITLE
Weather app pull request

### PR DESCRIPTION
**Pull request by Rashad-Muntar**
I implemented the functionalities below,
- I used the OpenWeather API to fetch the current weather condition of a location.
- The query returns a JSON format of the data which is then processed to get values using Asyn/Await ES6 syntax
- User is able to send a query  by using the form to get the weather condition of a city or location
-  User is able to toggle between Celcius and Fehrenheit.
-  There is a preloader that appears and disappears depending if our is successful or not

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Rashad-Muntar, congratulations on this project :+1: The user can search for a specific city to view its weather information :100: The use of promises or async/await has been utilized :1st_place_medal: The toggle functionality has partially been implemented :+1: 

However, there are a few comments that need to be addressed before proceeding. Kindly check the comments below:
- The toggle functionality should not only change the value but also the unit as well. Currently, it is only the value that changes. Kindly fix it
- The appearance of the app should change based on the received data. Currently, this functionality is missing. Let us address it

Once these are addressed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @Rashad-Muntar 👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'main.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.scss$/i,
+        use: ['style-loader', 'css-loader', 'sass-loader'],
+      },
+    ],
```
Comment (mikenath223) [inline webpack.config.js]: - [x] The project fails to compile with the error: `Error: Can't resolve '\weather-app\src\
  index.html'`. You could either update the directory structure and place index.html in `src` directory or update the ` webpack.config`.

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED**

Hello @Rashad-Muntar 
Great work making the requested changes.👏️👏️👏️
You can merge.👍️👍️

Happy coding💻️💻️

**Pull request merged**